CHANGLOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## [0.0.1] - 2025-03-11
```

Comment:
Please `s/0.0.1-SNAPSHOT/0.0.1/` in
https://github.com/nngrz/github-ai-bot/blob/main/pom.xml#L14C11-L14C25
Those two should match